remote-api/src/test/resources/alfresco-global.properties

```diff
@@ -1,4 +1,11 @@
 opencmis.tck.readtimeout=10000
 opencmis.tck.connecttimeout=10000
 
-system.workflow.engine.activiti.taskvariableslimit=200
\ No newline at end of file
+system.workflow.engine.activiti.taskvariableslimit=200
+
+system.directAccessUrl.enabled=true
+system.directAccessUrl.defaultExpiryTimeInSec=30
+system.directAccessUrl.maxExpiryTimeInSec=300
+
+restApi.directAccessUrl.enabled=true
+restApi.directAccessUrl.defaultExpiryTimeInSec=30
```

Comment:
I'm not sure we really want to enable DAUs on all other tests. Either use properties specifically for these tests, or mock the properties just for these tests (preferred)